lib/license_finder/package_manager.rb

```diff
@@ -69,7 +67,10 @@ def active?
       path = detected_package_path
       self.class.installed?(logger) &&
           !path.nil? &&
-        path.exist?.tap { |is_active| logger.active self.class, is_active }
+        path.exist?.tap do |is_active|
+          message = is_active ? Logger.green("is active") : 'is not active'
+          logger.log self.class, message
```

Comment:
Just double checking:
Seems like there is a Class method on `Logger` for `.green` and `.red`, but then in this method `logger` is an instance being passed in and `#log` is being called on it. Is that intentional?

My first impression implies that `Logger.green("Hello World")` would print out "Hello World" in green. Instead, it seems like it's colour coding the message. I feel like having that be a part of the `.log` signature rather than encoding the string with it is more intuitive.